src/sass/main.sass

```diff
@@ -74,7 +74,7 @@ body
             width: 100%
             height: 100%
             border: none
-            padding: 50px
+            padding: 0 10px 0 10px
         footer
             padding: 2%
```

Comment:
```suggestion
            
```

Reply:
If only adding padding without specifying it again have same alignment issue if you want I can change 10px to 2% for left and right

Reply:
No problem, I will merge. Thanks for contributing

Reply:
No problem looking forward to contributing more 🤞🏼